dev branch test failure: TestVolumeUnmount

Another intermittent test failure:
```
--- FAIL: TestVolumeUnmount (8.18s)
	docker_api_test_util.go:430: Created volume TestVolumeUnmount
	docker_api_test_util.go:261: Starting container: 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f
	docker_api_test_util.go:386: MQ is ready
	docker_api_test.go:297: Expected umount to work with rc=0, got 32
	docker_api_test_util.go:177: Inspected container 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f: types.ContainerJSON{ContainerJSONBase:(*types.ContainerJSONBase)(0xc42017f340), Mounts:[]types.MountPoint{types.MountPoint{Type:"bind", Name:"", Source:"/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage", Destination:"/var/coverage", Driver:"", Mode:"", RW:true, Propagation:"rprivate"}, types.MountPoint{Type:"volume", Name:"TestVolumeUnmount", Source:"/var/lib/docker/volumes/TestVolumeUnmount/_data", Destination:"/mnt/mqm", Driver:"local", Mode:"z", RW:true, Propagation:""}}, Config:(*container.Config)(0xc4202803c0), NetworkSettings:(*types.NetworkSettings)(0xc420279a00)}
	docker_api_test_util.go:182: Inspected container 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f: {
		    "Id": "0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f",
		    "Created": "2018-05-25T08:31:37.549557096Z",
		    "Path": "runmqdevserver",
		    "Args": [],
		    "State": {
		        "Status": "running",
		        "Running": true,
		        "Paused": false,
		        "Restarting": false,
		        "OOMKilled": false,
		        "Dead": false,
		        "Pid": 6519,
		        "ExitCode": 0,
		        "Error": "",
		        "StartedAt": "2018-05-25T08:31:38.104301858Z",
		        "FinishedAt": "0001-01-01T00:00:00Z"
		    },
		    "Image": "sha256:8b5eed066b58335c3033a6e4979557be2c1661812ee9f07a87108c74ba123649",
		    "ResolvConfPath": "/var/lib/docker/containers/0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f/resolv.conf",
		    "HostnamePath": "/var/lib/docker/containers/0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f/hostname",
		    "HostsPath": "/var/lib/docker/containers/0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f/hosts",
		    "LogPath": "/var/lib/docker/containers/0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f/0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f-json.log",
		    "Name": "/TestVolumeUnmount",
		    "RestartCount": 0,
		    "Driver": "overlay2",
		    "MountLabel": "",
		    "ProcessLabel": "",
		    "AppArmorProfile": "",
		    "ExecIDs": null,
		    "HostConfig": {
		        "Binds": [
		            "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage:/var/coverage",
		            "TestVolumeUnmount:/mnt/mqm"
		        ],
		        "ContainerIDFile": "",
		        "LogConfig": {
		            "Type": "json-file",
		            "Config": {}
		        },
		        "NetworkMode": "default",
		        "PortBindings": null,
		        "RestartPolicy": {
		            "Name": "",
		            "MaximumRetryCount": 0
		        },
		        "AutoRemove": false,
		        "VolumeDriver": "",
		        "VolumesFrom": null,
		        "CapAdd": [
		            "SYS_ADMIN"
		        ],
		        "CapDrop": null,
		        "Dns": null,
		        "DnsOptions": null,
		        "DnsSearch": null,
		        "ExtraHosts": null,
		        "GroupAdd": null,
		        "IpcMode": "shareable",
		        "Cgroup": "",
		        "Links": null,
		        "OomScoreAdj": 0,
		        "PidMode": "",
		        "Privileged": false,
		        "PublishAllPorts": false,
		        "ReadonlyRootfs": false,
		        "SecurityOpt": null,
		        "UTSMode": "",
		        "UsernsMode": "",
		        "ShmSize": 67108864,
		        "Runtime": "runc",
		        "ConsoleSize": [
		            0,
		            0
		        ],
		        "Isolation": "",
		        "CpuShares": 0,
		        "Memory": 0,
		        "NanoCpus": 0,
		        "CgroupParent": "",
		        "BlkioWeight": 0,
		        "BlkioWeightDevice": null,
		        "BlkioDeviceReadBps": null,
		        "BlkioDeviceWriteBps": null,
		        "BlkioDeviceReadIOps": null,
		        "BlkioDeviceWriteIOps": null,
		        "CpuPeriod": 0,
		        "CpuQuota": 0,
		        "CpuRealtimePeriod": 0,
		        "CpuRealtimeRuntime": 0,
		        "CpusetCpus": "",
		        "CpusetMems": "",
		        "Devices": null,
		        "DiskQuota": 0,
		        "KernelMemory": 0,
		        "MemoryReservation": 0,
		        "MemorySwap": 0,
		        "MemorySwappiness": null,
		        "OomKillDisable": false,
		        "PidsLimit": 0,
		        "Ulimits": null,
		        "CpuCount": 0,
		        "CpuPercent": 0,
		        "IOMaximumIOps": 0,
		        "IOMaximumBandwidth": 0
		    },
		    "GraphDriver": {
		        "Name": "overlay2",
		        "Data": {
		            "LowerDir": "/var/lib/docker/overlay2/f5e09ed10d303a1a7f1d59dcdc87898548f5e84465ebc30f4294d06b813be249-init/diff:/var/lib/docker/overlay2/f1574da8ec9925bd54e094e95fe064d79488f819e5af5fb840208e3bd9336855/diff:/var/lib/docker/overlay2/ff6f2808bbb89601a56ee79c50719feb367b29e127b5301c2be72c1d02064e48/diff:/var/lib/docker/overlay2/dd313fdde6c3b9e0354f187dab307a0223fae0c239be7ef6ff1192538bcb845c/diff:/var/lib/docker/overlay2/d5e7e35e3b63e9d52ad3213fb224c2ca29a7da635e60907f8357328eec7edf78/diff:/var/lib/docker/overlay2/002b4bb5ac02338c00afcd7c75f50a88504e817550ca987031aa24c8df71c4ba/diff:/var/lib/docker/overlay2/55e800f4bca870003716f05416fb76f03c681e5665df310fd2f89155ef5e1f51/diff:/var/lib/docker/overlay2/04d8812d37a009ee65945aa0ebde47b82cf1acbd8d42230d19b33f75b76976cf/diff:/var/lib/docker/overlay2/74bc8781b7c3c14ba2bc7fe92d486227200b98d25a73853093270673636b1767/diff:/var/lib/docker/overlay2/f6f923b402fd0c34afe2aec071ce3aeb641df813bbe38ab19a8748b5e75bf21e/diff:/var/lib/docker/overlay2/8039d3fb9c9bd1e0265ece4def4a5a3e8fb87ae1a5f9b2a7ee32f9eaf6604023/diff:/var/lib/docker/overlay2/b5f9f9321cbf0f19e0713b3646bd0b664a6cf4f46dc70607601fc155bc3f313b/diff:/var/lib/docker/overlay2/245f6ba61d08078696d7f23e1e8f58f385d6ac27a55412a665b5f3ea8827ea8f/diff:/var/lib/docker/overlay2/0d2d15e4085e40b74a417a26ef4713687242245281e26479ea571ede019b79ff/diff:/var/lib/docker/overlay2/af38f2a93336175e5e2c9bb0bf3521d2f3287d804b4aab7da5c1ea59331ea2eb/diff:/var/lib/docker/overlay2/33fe26ab63ded237a3e084df7232ee640d59f77784ad749c73b05e3156d02fa6/diff:/var/lib/docker/overlay2/c3009edef93a555e10946727fe7c3ddaa8c3ef1f364ebfb5323eae1b1521484a/diff:/var/lib/docker/overlay2/48792130de1aa9d38e9ba4639fd4be63a45c7030738d628af71a244506c8f775/diff:/var/lib/docker/overlay2/deaa02dbe914aa63cedaf4be93ef32ac59f6ad2e806ae8d88bb0de965cafe46e/diff:/var/lib/docker/overlay2/142500ed8c2c17d857f8b9d082762420cdb733041f50ab0abaa688ad464cb0c5/diff",
		            "MergedDir": "/var/lib/docker/overlay2/f5e09ed10d303a1a7f1d59dcdc87898548f5e84465ebc30f4294d06b813be249/merged",
		            "UpperDir": "/var/lib/docker/overlay2/f5e09ed10d303a1a7f1d59dcdc87898548f5e84465ebc30f4294d06b813be249/diff",
		            "WorkDir": "/var/lib/docker/overlay2/f5e09ed10d303a1a7f1d59dcdc87898548f5e84465ebc30f4294d06b813be249/work"
		        }
		    },
		    "Mounts": [
		        {
		            "Type": "bind",
		            "Source": "/home/travis/gopath/src/github.com/ibm-messaging/mq-container/test/docker/coverage",
		            "Destination": "/var/coverage",
		            "Mode": "",
		            "RW": true,
		            "Propagation": "rprivate"
		        },
		        {
		            "Type": "volume",
		            "Name": "TestVolumeUnmount",
		            "Source": "/var/lib/docker/volumes/TestVolumeUnmount/_data",
		            "Destination": "/mnt/mqm",
		            "Driver": "local",
		            "Mode": "z",
		            "RW": true,
		            "Propagation": ""
		        }
		    ],
		    "Config": {
		        "Hostname": "0d2fe4f5edbe",
		        "Domainname": "",
		        "User": "",
		        "AttachStdin": false,
		        "AttachStdout": false,
		        "AttachStderr": false,
		        "ExposedPorts": {
		            "1414/tcp": {},
		            "9443/tcp": {}
		        },
		        "Tty": false,
		        "OpenStdin": false,
		        "StdinOnce": false,
		        "Env": [
		            "LICENSE=accept",
		            "MQ_QMGR_NAME=qm1",
		            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
		            "LANG=en_US.UTF-8",
		            "AMQ_DIAGNOSTIC_MSG_SEVERITY=1",
		            "AMQ_ADDITIONAL_JSON_LOG=1",
		            "LOG_FORMAT=basic",
		            "MQ_DEV=true",
		            "MQ_ADMIN_PASSWORD=passw0rd"
		        ],
		        "Cmd": null,
		        "ArgsEscaped": true,
		        "Image": "mqadvanced-server-dev:9.0.5.0-x86_64-ubuntu-16.04",
		        "Volumes": null,
		        "WorkingDir": "",
		        "Entrypoint": [
		            "runmqdevserver"
		        ],
		        "OnBuild": null,
		        "Labels": {
		            "IBM_PRODUCT_ID": "98102d16795c4263ad9ca075190a2d4d",
		            "IBM_PRODUCT_NAME": "IBM MQ Advanced for Developers (Non-Warranted)",
		            "IBM_PRODUCT_VERSION": "9.0.5.0"
		        }
		    },
		    "NetworkSettings": {
		        "Bridge": "",
		        "SandboxID": "ac8035262c8c67d56719bba557852cdd9d2b26803b3956a47bb8625bc7e0ef6a",
		        "HairpinMode": false,
		        "LinkLocalIPv6Address": "",
		        "LinkLocalIPv6PrefixLen": 0,
		        "Ports": {
		            "1414/tcp": null,
		            "9443/tcp": null
		        },
		        "SandboxKey": "/var/run/docker/netns/ac8035262c8c",
		        "SecondaryIPAddresses": null,
		        "SecondaryIPv6Addresses": null,
		        "EndpointID": "d3411e83c4aecec2231fb0d6bf3a00465282b53050ee2e32bdac53dda6adf679",
		        "Gateway": "172.17.0.1",
		        "GlobalIPv6Address": "",
		        "GlobalIPv6PrefixLen": 0,
		        "IPAddress": "172.17.0.3",
		        "IPPrefixLen": 16,
		        "IPv6Gateway": "",
		        "MacAddress": "02:42:ac:11:00:03",
		        "Networks": {
		            "bridge": {
		                "IPAMConfig": null,
		                "Links": null,
		                "Aliases": null,
		                "NetworkID": "4b0bc2a4b5af56ea9f48b42bd5579d21e636a9e0c530f35c9bce50366389aefc",
		                "EndpointID": "d3411e83c4aecec2231fb0d6bf3a00465282b53050ee2e32bdac53dda6adf679",
		                "Gateway": "172.17.0.1",
		                "IPAddress": "172.17.0.3",
		                "IPPrefixLen": 16,
		                "IPv6Gateway": "",
		                "GlobalIPv6Address": "",
		                "GlobalIPv6PrefixLen": 0,
		                "MacAddress": "02:42:ac:11:00:03"
		            }
		        }
		    }
		}
	docker_api_test_util.go:184: Stopping container: 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f
	docker_api_test_util.go:192: Container stopped
	docker_api_test_util.go:197: Console log from container 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f:
		2018-05-25T08:31:38.102Z Set password for "admin" user
		2018-05-25T08:31:38.118Z Using queue manager name: qm1
		2018-05-25T08:31:38.118Z CPU architecture: amd64
		2018-05-25T08:31:38.118Z Linux kernel version: 4.4.0-101-generic
		2018-05-25T08:31:38.119Z Base image detected: Ubuntu 16.04.4 LTS
		2018-05-25T08:31:38.119Z Maximum file handles: 761443
		2018-05-25T08:31:38.119Z Running as user ID 0 (root) with primary group 0
		2018-05-25T08:31:38.119Z Detected capabilities: AUDIT_WRITE,CHOWN,DAC_OVERRIDE,FOWNER,FSETID,KILL,MKNOD,NET_BIND_SERVICE,NET_RAW,SETFCAP,SETGID,SETPCAP,SETUID,SYS_ADMIN,SYS_CHROOT
		2018-05-25T08:31:38.119Z Detected 'ext4' volume mounted to /mnt/mqm
		2018-05-25T08:31:38.119Z Detected /mnt/mqm has filesystem type 'ext'
		2018-05-25T08:31:38.138Z Created directory structure under /var/mqm
		2018-05-25T08:31:38.139Z Creating queue manager qm1
		2018-05-25T08:31:38.148Z Starting web server
		2018-05-25T08:31:41.795Z Starting queue manager
		2018-05-25T08:31:39.076Z AMQ6287I: IBM MQ V9.0.5.0 (p905-L180306.DE).
		2018-05-25T08:31:39.073Z AMQ5051I: The queue manager task 'LOGGER-IO' has started.
		2018-05-25T08:31:39.131Z AMQ5041I: The queue manager task 'LOGGER-IO' has ended.
		2018-05-25T08:31:39.494Z AMQ5051I: The queue manager task 'LOGGER-IO' has started.
		2018-05-25T08:31:39.520Z AMQ7229I: 4 log records accessed on queue manager 'qm1' during the log replay phase.
		2018-05-25T08:31:39.520Z AMQ7230I: Log replay for queue manager 'qm1' complete.
		2018-05-25T08:31:39.522Z AMQ7231I: 0 log records accessed on queue manager 'qm1' during the recovery phase.
		2018-05-25T08:31:39.524Z AMQ7232I: Transaction manager state recovered for queue manager 'qm1'.
		2018-05-25T08:31:39.542Z AMQ7233I: 0 out of 0 in-flight transactions resolved for queue manager 'qm1'.
		2018-05-25T08:31:39.544Z AMQ5051I: The queue manager task 'CHECKPOINT' has started.
		2018-05-25T08:31:39.664Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.678Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.678Z AMQ5037I: The queue manager task 'ERROR-LOG' has started.
		2018-05-25T08:31:39.678Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.663Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.663Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.663Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.681Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:39.684Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:41.438Z AMQ8048I: Default objects statistics : 85 created. 0 replaced. 0 failed.
		2018-05-25T08:31:41.445Z AMQ8003I: IBM MQ queue manager 'qm1' started using V9.0.5.0.
		2018-05-25T08:31:41.537Z AMQ5041I: The queue manager task 'APP-SIGNAL' has ended.
		2018-05-25T08:31:41.537Z AMQ5041I: The queue manager task 'APP-SIGNAL' has ended.
		2018-05-25T08:31:41.581Z AMQ5041I: The queue manager task 'ERROR-LOG' has ended.
		2018-05-25T08:31:41.634Z AMQ5041I: The queue manager task 'CHECKPOINT' has ended.
		2018-05-25T08:31:41.671Z AMQ5041I: The queue manager task 'LOGGER-IO' has ended.
		2018-05-25T08:31:41.749Z AMQ8004I: IBM MQ queue manager 'qm1' ended.
		2018-05-25T08:31:42.092Z AMQ5051I: The queue manager task 'LOGGER-IO' has started.
		2018-05-25T08:31:42.160Z AMQ7229I: 5 log records accessed on queue manager 'qm1' during the log replay phase.
		2018-05-25T08:31:42.161Z AMQ7230I: Log replay for queue manager 'qm1' complete.
		2018-05-25T08:31:42.166Z AMQ7231I: 0 log records accessed on queue manager 'qm1' during the recovery phase.
		2018-05-25T08:31:42.166Z AMQ7232I: Transaction manager state recovered for queue manager 'qm1'.
		2018-05-25T08:31:42.180Z AMQ7233I: 0 out of 0 in-flight transactions resolved for queue manager 'qm1'.
		2018-05-25T08:31:42.202Z AMQ5051I: The queue manager task 'CHECKPOINT' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.310Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.310Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.312Z AMQ5037I: The queue manager task 'APP-SIGNAL' has started.
		2018-05-25T08:31:42.329Z AMQ5037I: The queue manager task 'ERROR-LOG' has started.
		2018-05-25T08:31:42.431Z AMQ8003I: IBM MQ queue manager 'qm1' started using V9.0.5.0.
		2018-05-25T08:31:42.438Z AMQ5037I: The queue manager task 'DEFERRED_DELIVERY' has started.
		2018-05-25T08:31:42.446Z AMQ9410I: Repository manager started.
		2018-05-25T08:31:42.477Z AMQ5037I: The queue manager task 'DEFERRED-MSG' has started.
		2018-05-25T08:31:42.489Z AMQ5037I: The queue manager task 'STATISTICS' has started.
		2018-05-25T08:31:42.510Z AMQ5051I: The queue manager task 'ACTVTRC' has started.
		2018-05-25T08:31:42.517Z AMQ5051I: The queue manager task 'ASYNCQ' has started.
		2018-05-25T08:31:42.520Z AMQ5051I: The queue manager task 'EXPIRER' has started.
		2018-05-25T08:31:42.528Z AMQ5051I: The queue manager task 'DUR-SUBS-MGR' has started.
		2018-05-25T08:31:42.561Z AMQ5037I: The queue manager task 'MARKINTSCAN' has started.
		2018-05-25T08:31:42.587Z AMQ5052I: The queue manager task 'QPUBSUB-CTRLR' has started.
		2018-05-25T08:31:42.589Z AMQ5052I: The queue manager task 'QPUBSUB-QUEUE-NLCACHE' has started.
		2018-05-25T08:31:42.597Z AMQ5052I: The queue manager task 'QPUBSUB-SUBPT-NLCACHE' has started.
		2018-05-25T08:31:42.640Z AMQ5051I: The queue manager task 'TOPIC-TREE' has started.
		2018-05-25T08:31:42.641Z AMQ5051I: The queue manager task 'RESOURCE_MONITOR' has started.
		2018-05-25T08:31:42.642Z AMQ5052I: The queue manager task 'PUBSUB-DAEMON' has started.
		2018-05-25T08:31:42.642Z AMQ5975I: 'IBM MQ Distributed Pub/Sub Controller' has started.
		2018-05-25T08:31:42.643Z AMQ5051I: The queue manager task 'Q-DELETION' has started.
		2018-05-25T08:31:42.644Z AMQ5051I: The queue manager task 'PRESERVED-Q' has started.
		2018-05-25T08:31:42.653Z AMQ5051I: The queue manager task 'MULTICAST' has started.
		2018-05-25T08:31:42.670Z AMQ5975I: 'IBM MQ Distributed Pub/Sub Fan Out Task' has started.
		2018-05-25T08:31:42.681Z AMQ5975I: 'IBM MQ Distributed Pub/Sub Command Task' has started.
		2018-05-25T08:31:42.690Z AMQ5975I: 'IBM MQ Distributed Pub/Sub Publish Task' has started.
		2018-05-25T08:31:42.802Z AMQ5022I: The channel initiator has started. ProcessId(543).
		2018-05-25T08:31:42.803Z AMQ5024I: The command server has started. ProcessId(544).
		2018-05-25T08:31:42.916Z Started queue manager
		2018-05-25T08:31:42.804Z AMQ5026I: The listener 'SYSTEM.LISTENER.TCP.1' has started. ProcessId(545).
		2018-05-25T08:31:42.951Z AMQ5806I: Queued Publish/Subscribe Daemon started for queue manager qm1.
		2018-05-25T08:31:43.081Z AMQ8024I: IBM MQ channel initiator started.
		2018-05-25T08:31:43.116Z AMQ5027I: The listener 'SYSTEM.LISTENER.TCP.1' has ended. ProcessId(545).
		2018-05-25T08:31:43.415Z Output for "runmqsc" with /etc/mqm/10-dev.mqsc:
			5724-H72 (C) Copyright IBM Corp. 1994, 2018.
			Starting MQSC for queue manager qm1.
			
			
			       : * © Copyright IBM Corporation 2017, 2018
			       : *
			       : *
			       : * Licensed under the Apache License, Version 2.0 (the "License");
			       : * you may not use this file except in compliance with the License.
			       : * You may obtain a copy of the License at
			       : *
			       : * http://www.apache.org/licenses/LICENSE-2.0
			       : *
			       : * Unless required by applicable law or agreed to in writing, software
			       : * distributed under the License is distributed on an "AS IS" BASIS,
			       : * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
			       : * See the License for the specific language governing permissions and
			       : * limitations under the License.
			       : 
			     1 : STOP LISTENER('SYSTEM.LISTENER.TCP.1')
			AMQ8706I: Request to stop IBM MQ Listener accepted.
			       : 
			       : * Developer queues
			     2 : DEFINE QLOCAL('DEV.QUEUE.1') REPLACE
			AMQ8006I: IBM MQ queue created.
			     3 : DEFINE QLOCAL('DEV.QUEUE.2') REPLACE
			AMQ8006I: IBM MQ queue created.
			     4 : DEFINE QLOCAL('DEV.QUEUE.3') REPLACE
			AMQ8006I: IBM MQ queue created.
			     5 : DEFINE QLOCAL('DEV.DEAD.LETTER.QUEUE') REPLACE
			AMQ8006I: IBM MQ queue created.
			       : 
			       : * Use a different dead letter queue, for undeliverable messages
			     6 : ALTER QMGR DEADQ('DEV.DEAD.LETTER.QUEUE')
			AMQ8005I: IBM MQ queue manager changed.
			       : 
			       : * Developer topics
			     7 : DEFINE TOPIC('DEV.BASE.TOPIC') TOPICSTR('dev/') REPLACE
			AMQ8690I: IBM MQ topic created.
			       : 
			       : * Developer connection authentication
			     8 : DEFINE AUTHINFO('DEV.AUTHINFO') AUTHTYPE(IDPWOS) CHCKCLNT(REQDADM) CHCKLOCL(OPTIONAL) ADOPTCTX(YES) REPLACE
			AMQ8563I: IBM MQ authentication information object created.
			     9 : ALTER QMGR CONNAUTH('DEV.AUTHINFO')
			AMQ8005I: IBM MQ queue manager changed.
			    10 : REFRESH SECURITY(*) TYPE(CONNAUTH)
			AMQ8560I: IBM MQ security cache refreshed.
			       : 
			       : * Developer channels (Application + Admin)
			       : * Developer channels (Application + Admin)
			    11 : DEFINE CHANNEL('DEV.ADMIN.SVRCONN') CHLTYPE(SVRCONN) REPLACE
			AMQ8014I: IBM MQ channel created.
			    12 : DEFINE CHANNEL('DEV.APP.SVRCONN') CHLTYPE(SVRCONN) MCAUSER('app') REPLACE
			AMQ8014I: IBM MQ channel created.
			       : 
			       : * Developer channel authentication rules
			    13 : SET CHLAUTH('*') TYPE(ADDRESSMAP) ADDRESS('*') USERSRC(NOACCESS) DESCR('Back-stop rule - Blocks everyone') ACTION(REPLACE)
			AMQ8877I: IBM MQ channel authentication record set.
			    14 : SET CHLAUTH('DEV.APP.SVRCONN') TYPE(ADDRESSMAP) ADDRESS('*') USERSRC(CHANNEL) CHCKCLNT(ASQMGR) DESCR('Allows connection via APP channel') ACTION(REPLACE)
			AMQ8877I: IBM MQ channel authentication record set.
			    15 : SET CHLAUTH('DEV.ADMIN.SVRCONN') TYPE(BLOCKUSER) USERLIST('nobody') DESCR('Allows admins on ADMIN channel') ACTION(REPLACE)
			AMQ8877I: IBM MQ channel authentication record set.
			    16 : SET CHLAUTH('DEV.ADMIN.SVRCONN') TYPE(USERMAP) CLNTUSER('admin') USERSRC(CHANNEL) DESCR('Allows admin user to connect via ADMIN channel') ACTION(REPLACE)
			AMQ8877I: IBM MQ channel authentication record set.
			       : 
			       : * Developer authority records
			    17 : SET AUTHREC GROUP('mqclient') OBJTYPE(QMGR) AUTHADD(CONNECT,INQ)
			AMQ8862I: IBM MQ authority record set.
			    18 : SET AUTHREC PROFILE('DEV.**') GROUP('mqclient') OBJTYPE(QUEUE) AUTHADD(BROWSE,GET,INQ,PUT)
			AMQ8862I: IBM MQ authority record set.
			    19 : SET AUTHREC PROFILE('DEV.**') GROUP('mqclient') OBJTYPE(TOPIC) AUTHADD(PUB,SUB)
			AMQ8862I: IBM MQ authority record set.
			       : 
			       : * Developer listener
			    20 : DEFINE LISTENER('DEV.LISTENER.TCP') TRPTYPE(TCP) PORT(1414) CONTROL(QMGR) REPLACE
			AMQ8626I: IBM MQ listener created.
			    21 : START LISTENER('DEV.LISTENER.TCP')
			AMQ8021I: Request to start IBM MQ listener accepted.
			21 MQSC commands read.
			No commands have a syntax error.
			All valid MQSC commands were processed.
			
		2018-05-25T08:31:43.415Z Metrics are disabled
		2018-05-25T08:31:43.392Z AMQ5026I: The listener 'DEV.LISTENER.TCP' has started. ProcessId(628).
		2018-05-25T08:31:44.099Z Error 22 starting web server: Starting server mqweb.
		Server mqweb start failed. Check server logs for details.
		
		2018-05-25T08:31:44.817Z Signal received: terminated
		2018-05-25T08:31:44.817Z Stopping queue manager
		2018-05-25T08:31:44.833Z Error stopping queue manager: AMQ8101S: IBM MQ error (893) has occurred.
		
		2018-05-25T08:31:45.296Z stat /var/mqm/errors/AMQERR01.json: no such file or directory
		2018-05-25T08:31:45.308Z stat /var/mqm/qmgrs/qm1/errors/AMQERR01.json: no such file or directory
	docker_api_test_util.go:161: open /tmp/TestVolumeUnmount-termination-log: no such file or directory
	docker_api_test_util.go:205: Removing container: 0d2fe4f5edbee6bb5341b1db05ba9bf4563950fa90eebc726bad320a62e5866f
	docker_api_test_util.go:435: Removing volume TestVolumeUnmount
FAIL
exit status 1
FAIL	github.com/ibm-messaging/mq-container/test/docker	176.102s
make: *** [test-devserver] Error 1
```

--- a/test/docker/docker_api_test.go
+++ b/test/docker/docker_api_test.go
@@ -292,9 +292,9 @@ func TestVolumeUnmount(t *testing.T) {
 	defer cleanContainer(t, cli, ctr.ID)
 	waitForReady(t, cli, ctr.ID)
 	// Unmount the volume as root
-	rc, _ := execContainer(t, cli, ctr.ID, "root", []string{"umount", "-l", "-f", "/mnt/mqm"})
+	rc, out := execContainer(t, cli, ctr.ID, "root", []string{"umount", "-l", "-f", "/mnt/mqm"})
 	if rc != 0 {
-		t.Fatalf("Expected umount to work with rc=0, got %v", rc)
+		t.Fatalf("Expected umount to work with rc=0, got %v. Output was: %s", rc, out)
 	}
 	time.Sleep(3 * time.Second)
 	rc, _ = execContainer(t, cli, ctr.ID, "mqm", []string{"chkmqhealthy"})
@@ -329,7 +329,9 @@ func TestZombies(t *testing.T) {
 	// will be adopted by PID 1, and should then be reaped when they die.
 	_, out := execContainer(t, cli, id, "mqm", []string{"pkill", "--signal", "kill", "-c", "amqzxma0"})
 	if out == "0" {
-		t.Fatalf("Expected pkill to kill a process, got %v", out)
+		t.Log("Failed to kill process 'amqzxma0'")
+		_, out := execContainer(t, cli, id, "root", []string{"ps", "-lA"})
+		t.Fatalf("Here is a list of currently running processes:\n%s", out)
 	}
 	time.Sleep(3 * time.Second)
 	_, out = execContainer(t, cli, id, "mqm", []string{"bash", "-c", "ps -lA | grep '^. Z'"})

--- a/test/docker/docker_api_test_util.go
+++ b/test/docker/docker_api_test_util.go
@@ -27,11 +27,11 @@ import (
 	"log"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
-	"regexp"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
@@ -257,7 +257,6 @@ func waitForContainer(t *testing.T, cli *client.Client, ID string, timeout int64
 
 // execContainer runs a command in a running container, and returns the exit code and output
 func execContainer(t *testing.T, cli *client.Client, ID string, user string, cmd []string) (int, string) {
-	rerun:
 	config := types.ExecConfig{
 		User:        user,
 		Privileged:  false,
@@ -283,30 +282,33 @@ func execContainer(t *testing.T, cli *client.Client, ID string, user string, cmd
 	})
 	// Wait for the command to finish
 	var exitcode int
+	var outputStr string
 	for {
 		inspect, err := cli.ContainerExecInspect(context.Background(), resp.ID)
 		if err != nil {
 			t.Fatal(err)
 		}
-		if !inspect.Running {
-			exitcode = inspect.ExitCode
-			break
+		if inspect.Running {
+			continue
 		}
-	}
-	buf := new(bytes.Buffer)
-	// Each output line has a header, which needs to be removed
-	_, err = stdcopy.StdCopy(buf, buf, hijack.Reader)
-	if err != nil {
-		log.Fatal(err)
-	}
 
-	outputStr := strings.TrimSpace(buf.String())
+		exitcode = inspect.ExitCode
+		buf := new(bytes.Buffer)
+		// Each output line has a header, which needs to be removed
+		_, err = stdcopy.StdCopy(buf, buf, hijack.Reader)
+		if err != nil {
+			t.Fatal(err)
+		}
 
-	// Before we go let's just double check it did actually run because sometimes we get a "Exec command already running error"
-	alreadyRunningErr := regexp.MustCompile("Error: Exec command .* is already running")
-	if alreadyRunningErr.MatchString(outputStr) {
-		time.Sleep(1 * time.Second)
-		goto rerun
+		outputStr = strings.TrimSpace(buf.String())
+
+		// Before we go let's just double check it did actually finish running
+		// because sometimes we get a "Exec command already running error"
+		alreadyRunningErr := regexp.MustCompile("Error: Exec command .* is already running")
+		if alreadyRunningErr.MatchString(outputStr) {
+			continue
+		}
+		break
 	}
 
 	return exitcode, outputStr

